new-dti-website-redesign/src/components/TeamCard.tsx

```diff
@@ -0,0 +1,243 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { getColorClass, productLinks } from '../utils/memberUtils';
+import Button from './Button';
+
+const MemberSummary = ({
+  user,
+  image,
+  enlarged = false
+}: {
+  user: IdolMember;
+  image: string;
+  enlarged?: boolean;
+}) => (
+  <>
+    <Image
+      src={image}
+      alt={`${user.firstName} ${user.lastName}'s profile picture`}
+      className="rounded-lg w-auto h-auto"
+      width={232}
+      height={232}
+    />
+    <div className="text-left">
+      {enlarged ? (
+        <>
+          <h3>{`${user.firstName} ${user.lastName}`}</h3>
+          <h5 className={`text-${getColorClass(user.role)}`}>{user.roleDescription}</h5>
+        </>
+      ) : (
+        <>
+          <h6>{`${user.firstName} ${user.lastName}`}</h6>
+          <p className={`text-${getColorClass(user.role)}`}>{user.roleDescription}</p>
+        </>
+      )}
+    </div>
+  </>
+);
+
+type MemberCardProps = {
+  user: IdolMember;
+  image: string;
+  selected: boolean;
+  onClick?: () => void;
+};
+
+export const MemberCard = ({ user, image, selected, onClick }: MemberCardProps) => {
+  const baseStyles =
+    'relative p-8 border border-border-1 flex flex-col gap-4 hover:bg-background-2 transition-[background-color] duration-[120ms] has-[:focus]:outline-2 has-[:focus]:outline-offset-3 has-[:focus]:z-10';
+
+  return (
+    <article
+      className={`${baseStyles} ${selected ? `bg-background-2 border-b-foreground-1 after:content-[''] after:absolute after:bottom-[-1px] after:left-0 after:w-full after:h-[1px] after:bg-foreground-1 after:shadow-[0_-4px_8px_0_var(--foreground-1)] after:rounded-full` : ''}`}
+    >
+      <MemberSummary user={user} image={image} />
+      <button
+        className="opacity-0 cursor-pointer after:content-[''] after:absolute after:top-0 after:left-0 after:w-full after:h-full focus:outline-4"
+        onClick={onClick}
+      />
+    </article>
+  );
+};
+
+type MemberDetailsProps = {
+  user: IdolMember;
+  image: string;
+};
+export const MemberDetailsCard = ({ user, image }: MemberDetailsProps) => {
+  const baseStyles = 'md:w-1/2 border border-border-1 flex flex-col';
+  return (
+    <div className="flex w-full flex-col md:flex-row">
+      <div className={`${baseStyles} p-8 gap-4`}>
```

Comment:
```suggestion
      <div className={`${baseStyles} p-8 gap-4 border-r-0`}>
```

To remove the double border on the right side of the left column